### ISSUE TITLE
Cannot install dada2

### ISSUE THREAD

**Issue by whyempty89** [benjjneb/dada2#1479]
Hello there,

I've followed the installation guide step-by-step, but am unable to install dada2. My main problem, it seems to me anyway, is that I cannot download/install the latest version of "fansi" and "GenomeInfoDbData". Whatever code I try, it always leads to the window that will ask me "Do you want to install from sources the packages which needs compilation"? I go with us, but these 2 packages always result in "non-zero sum exit" error. So I thought maybe I've downloaded Rtools wrongly, which is why my packages are not compiling properly. But I have uninstalled & reinstalled Rtools, again following the instructions to create the .Renviron file, and everything seems to be fine. I'm also getting an additional error of  
"
Error: Failed to install 'dada2' from GitHub:
  System command 'Rcmd.exe' failed, exit status: 1, stdout & stderr were printed
"
I can install "fansi" using type = "binary", but that obviously doesn't help that dada2 installation. And GenomeInfoDbData just seems impossible to download. 

I don't know what the problem is which is really frustrating as I'd really like to try to use dada2 in R instead of Qiime2. These errors are also preventing me from installing phyloseq, so I then can't even use my Qiime2 outputs in R anyway. 

I have tried Googling my errors, but nothing seems to work and out of desperation I'm posting to the dada2 issue tracker. I would really appreciate any help I can get on this issue.

**Comment by benjjneb** [comment]
`GenomeInfoDbData` has been a problem in the past, especially for folks with non-standard setups. Can you say a little bit more about your machine/OS including versions? I'm guessing it is Windows because of Rtools? But what version of Windows? Do you have admin access to your machine?

You may want to ask the question about installing `GenomeInfoDbData` at the Bioconductor help forum as well: https://support.bioconductor.org/ 

Because I think if you can get that installed, then dada2 will install fine, it's just faililng on that dependency.

**Comment by whyempty89** [comment]
Thanks for replying @benjjneb 

I'm running on a Windows 10, and my R is 4.1.2 and Rtools 4.0. I do have admin access to my machine. 

I was trying to troubleshoot this with a colleague the other day. We uninstalled and reinstalled R, Rtools, Rstudio several times. Checked that the .Renviron file was in the right place. Unsync my R from the OneDrive folder (we read somewhere R/Rtools doesn't like working out of cloud storage folders). None of this worked. It wasn't just GenomeInfoDbData that I have problems installing, I couldn't even install ggplot2 from source (when I install.packages via type = "source"). Then tried installing dada2 and Phyloseq on 2 other machines and had no problems whatsoever.

I _think_ the issue I'm having here is my machine, which has been giving me quite a few problems over the past 3.5 years. Sometime last year, it decided it no longer wanted to run .cmd files. I would click on it, a command prompt window would pop up, then immediately disappear. So I thought maybe this inability to run .cmd files is not allowing Rtools to compile my packages? Not entirely sure, but it's the only thing I can think of currently. Not the best problem, since if it _is_ the problem, the only fix is to fix the machine (which I've tried. Our IT department was unable to help). But also, I kind of hope it is the problem because at least I would have figure out why I can't get Rtools to work.
